api/models/HDescription.js

```diff
@@ -8,7 +8,7 @@
 module.exports = {
   tableName: 'h_description',
 
-  primaryKey: 'id',
+  primaryKey: 'dateReviewed',
```

Comment:
What is the reason of changing this ?
It seems that the primaryKey property is only used by waterline in "migration" -> alter mode but in our case we use the safe mode to prevent waterline to do any change automatically to our schema...
SailsJS doc discourage to change "id" with something else : 
![image](https://user-images.githubusercontent.com/985903/211687301-34841f6b-cff4-49ec-abe1-7f1950032bc9.png)

And composite primary key (as it is the case for HDescription is still not supported by Waterline : https://github.com/balderdashy/sails/issues/6359

Reply:
If we put id as the primary key, as it is supposed to be unique, the .find of sails retrieves only one value from the table while there can be multiple. As it is not possible to make composite primary keys with sails, this is why we had put the other attribute of the primary key of h_**** which is dateReviewed and which is unique.

Reply:
We find a way to avoid using `primary_key:date_reviewed` by changing the model. 
`id` now represent `date_reviewed` because it is unique
`id_location` represent `id`